src/transformState.js

```diff
@@ -62,7 +62,21 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const obj of transforms) {
```

Comment:
```suggestion
  for (const { operation, properties } of transforms) {
```
let's destructure it

Reply:
Sure I will change it, thanks!